templates/module/helfi-news-item/node--news-item.html.twig

```diff
@@ -0,0 +1,78 @@
+{% embed '@hdbt/content/node.html.twig' with {'node_class': 'components components--news'} %}
+
+  {% block main_content %}
+
+    {# Lead text #}
+    {% if content.field_lead_in|render %}
+      <section class="preamble">
```

Comment:
Why different class name for the lead-in paragraph here and in basic page?

Reply:
Is it the after pseudo element that we want here but not on basic page? I would have in that case used a modifier here like lead-in--news-item or something similar.

Reply:
The reasoning is that the old lead-in is so complex and used in several places, I wanted something that we could start building from the ground up.

I think we need to unify the different page styles to a more common ground instead of doing these one-offs.

It could also be something like leadin_v2